<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>680</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    font: 9pt &quot;微软雅黑&quot;;
}
#widgetTitle{
    background-color: rgba(0, 0, 0, 16);
}
#widgetPlayBar{
    background-color: rgba(0, 0, 0, 64);
}
QLabel
{
    color: #FFFFFF;
}
/* === QPushButton === */
QPushButton {
    border: 1px solid #444444;
    padding: 4px;
    color: #FFFFFF;
    background-color: #222222;
}
 
QPushButton:hover {
    background-color: #333333;
    border-color: #444444;
}
 
QPushButton:pressed {
    background-color: #111111;
    border-color: #333333;
    color: yellow;
}
 
QPushButton:disabled {
    color: #333333;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QTabLineGLWidget" name="widgetPlayer" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetTitle" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelTitle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/image/resource/image/logo_26.png);
background-repeat: no;</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="indent">
             <number>32</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonMin">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
    width: 26px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_min.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_min.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_min.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonMax">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
    width: 26px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_max.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_max.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_max.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClose">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
    width: 26px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_close.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_close.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_close.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widgetPlayBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">#widgetPlayBar{
background-color: rgba(0, 0, 0, 64);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonOpen">
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Open</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_open.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_open.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_open.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLeft">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Previous tab</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_prev.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_prev.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_prev.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPlay">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Play/Pause</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_play.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_play.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_play.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRight">
            <property name="minimumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Next tab</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: 0px;
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_next.png) 0 52 0 0;
}
 
QPushButton:hover {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_next.png) 0 26 0 26;
}
 
QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0);
	border-image: url(:/image/resource/image/btn_next.png) 0 0 0 52;
}
 
QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="progress">
            <property name="styleSheet">
             <string notr="true">QSlider::groove:horizontal {
    border: 0px;
    height: 4px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #212121, stop:1 #444444);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 8px;
    margin: -2px 0;
    border-radius: 3px;
}
QSlider::add-page:vertical {
  background: white;
}

QSlider::sub-page:vertical {
  background: pink;
}</string>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineHeightComboBox" name="comboBoxLineHeight">
            <property name="toolTip">
             <string>Tab line height</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSlowDown">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Slow down</string>
            </property>
            <property name="text">
             <string>↓</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnResetSpeed">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Reset speed</string>
            </property>
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSpeedUp">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Speed up</string>
            </property>
            <property name="text">
             <string>↑</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTabLineGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">qtablineglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLineHeightComboBox</class>
   <extends>QComboBox</extends>
   <header>qlineheightcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
